doc/nrf/libraries/others/dm.rst

```diff
@@ -0,0 +1,70 @@
+.. _mod_dm:
+
+Distance Measurement
+####################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Distance Measurement module provides an integration of :ref:`nrfxlib:nrf_dm` into |NCS|.
+
+Overview
+********
+
+The Distance Measurement module is independent of the communication protocol used.
+The :ref:`nrfxlib:mpsl` is used to access the radio module which is required by the :ref:`nrfxlib:nrf_dm` library.
+During the allocated tiemslot, a distance measurement is performed.
+
+Using Distance Measurement
+**************************
+
+To use Distance Measurement module, enable it using the :kconfig:`CONFIG_NRF_DM` Kconfig option and initialize it in your :file:`main.c` file.
+After synchronizing the nodes between which the measurement is to be performed, add the measurement request.
+When the measurement data are available, the callback function will be called.
+
+Complete the following steps:
+
+1. Enable the :kconfig:`CONFIG_NRF_DM` Kconfig option.
+#. Include :file:`dm.h` in your :file:`main.c` file.
+#. Call :c:func:`dm_init()` to initialize the module.
+#. Call :c:func:`dm_request_add()` to take a measurement.
+
+Configuration
+*************
+
+To use the Distance Measurement module, enable the :kconfig:`CONFIG_NRF_DM` Kconfig option.
+
+Adjusting nodes synchronization can be accomplished by changing the values of the parameters: :kconfig:`CONFIG_DM_INITIATOR_DELAY_US`, :kconfig:`CONFIG_DM_REFLECTOR_DELAY_US` and :kconfig:`CONFIG_DM_MIN_TIME_BETWEEN_TIMESLOTS_US`.
+
+The parameter :kconfig:`CONFIG_DM_RANGING_OFFSET_US` is the time between synchronization (adding a request) and the rangigng.
+Increasing this allows for more rangings to different nodes but also increases latency.
+
+Enable the :kconfig:`CONFIG_DM_GPIO_DEBUG` parameter causes the pins state to change when a new measurement request is added and when the timeslot is assigned.
+This functionality can be used to determine synchronization accuracy.
+Assignment of a pin number is realized by using :kconfig:`CONFIG_DM_RANGING_PIN` and :kconfig:`CONFIG_DM_ADD_REQUEST_PIN`.
+
+You can set the following Kconfig options:
+   * :kconfig:`CONFIG_NRF_DM_TIMER`
```

Comment:
```suggestion
You can set the following Kconfig options:

* :kconfig:`CONFIG_NRF_DM_TIMER`
```
Align the rest of the options with the first one here.

Reply:
Fixed.